code/__HELPERS/text.dm

```diff
@@ -809,6 +809,31 @@ GLOBAL_LIST_INIT(binary, list("0","1"))
 		else
 			return "[number]\th"
 
+/proc/numtotext(number)
```

Comment:
Should specify either in the name (inttotext?) or the autodoc that this only supports integers